packages/devtools_app/lib/src/theme.dart

```diff
@@ -160,22 +160,23 @@ const defaultElevation = 4.0;
 const borderPadding = 2.0;
 const densePadding = 4.0;
 
-const smallProgressSize = 12.0;
+double get smallProgressSize => scaleByFontFactor(12.0);
 
-const defaultListItemHeight = 28.0;
+double get defaultListItemHeight => scaleByFontFactor(28.0);
 
-const defaultChartHeight = 150.0;
+double get defaultChartHeight => scaleByFontFactor(150.0);
```

Comment:
I'm not sure we should be scaling this. For example, the flutter frames chart and the memory chart that use this value for their height do not have any text in them that would be affected by font factor